manifest.json

```diff
@@ -415,6 +415,13 @@
         "url": "https://github.com/yui/yuicompressor/releases/download/v2.4.8",
         "files": ["yuicompressor-2.4.8.jar"],
         "targetDirPrefix": "yuicompressor-"
+      },
+      "browsermob-proxy": {
```

Comment:
No. Every library should be installed in only one location.

Though it's weird that one of these appears to be 2.1.1 and the other 0.7.1...